src/Tribe/Commerce/PayPal/Main.php

```diff
@@ -2260,19 +2262,28 @@ public function maybe_delete_expired_products() {
 
 		$transient_key = $this->get_current_cart_transient();
 
+		// Bail if we have no data key.
 		if ( false === $transient_key ) {
 			return;
 		}
 
-		delete_transient( $transient_key );
+		$transient = get_transient( $transient_key );
+
+		// Bail if we have no data to delete.
+		if ( empty( $transient ) ) {
+			return;
+		}
 
 		// Bail if ET+ is not in place.
 		if ( ! class_exists( 'Tribe__Tickets_Plus__Meta__Storage' ) ) {
 			return;
 		}
-
 		$storage = new Tribe__Tickets_Plus__Meta__Storage();
-		$storage->delete_cookie();
```

Comment:
This will fail every time we get here - `delete_cookie()` requires a passed ticket id.